ding2.profile

```diff
@@ -658,9 +658,17 @@ function ding2_module_list_as_operations($module_list) {
  */
 function ding2_module_enable(&$install_state) {
   $modules = variable_get('ding_module_selected', array());
-  $modules[] = 'l10n_update';
-  $modules[] = 'ting_infomedia';
-  $modules[] = 'ding_eresource';
+  // Modules we dont have an explicit dependency on but still want enabled by
+  // default. If the user later on does not need the module it can be disabled
+  // manually.
+  $modules = array(
```

Comment:
:+1: I will fix this.

We have added new modules here which depend directly on OpenSearch and were previously enabled by default as they were dependencies of other modules. To allow sites working with other search providers we have made these module non-required. Instead we enable them here for new installations.

> The functionality of some modules are directly tied to Open* systems. These have been updated to use opensearch.client.inc to signal this dependency instead of calling ting.client.inc or the search provider. Such modules have also been removed as hard dependencies and are instead enabled on install time.